plugins/bundle/config_test.go

```diff
@@ -340,6 +338,14 @@ b3:
 			service:  "s3",
 			resource: "/some/longer/path/bundle.tar.gz",
 		},
+		"b4": {
+			service:  "s1",
```

Comment:
Why would this be `s1`? 🤔 Is that the default because no service was provided in [the config below `b4`](https://github.com/open-policy-agent/opa/pull/3429/commits/7bfb804975d4022b5723d4f3f22a8af1bd32d129#diff-f1d9625ec0db3e9c4594fab73d5880b226c8ead819efcf98bdceaf9a3b89bccfR313)?

Reply:
The service will default to the first one in the list.